tests/test_other.py

```diff
@@ -7339,14 +7339,6 @@ def test_binaryen_invalid_method(self):
     proc.communicate()
     assert proc.returncode != 0
 
-  def test_binaryen_default_method(self):
```

Comment:
i'm ok to remove this test (together with the logging removed in this pr) but it does verify that we emit native wasm by default, instead of the interpreter. i think it's worth verifying that. we could test it in another way, perhaps, but no great way seems obvious. maybe it's enough to see that when enabling the interpreter, the js in the build is massively bigger.